launch/launch/actions/set_environment_variable.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2018 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for the SetEnvironmentVariable action."""
+
+import os
+from typing import List
+
+from ..action import Action
+from ..launch_context import LaunchContext
+from ..some_substitutions_type import SomeSubstitutionsType
+from ..substitution import Substitution
+from ..utilities import normalize_to_list_of_substitutions
+from ..utilities import perform_substitutions
+
+
+class SetEnvironmentVariable(Action):
+    """ Action that sets an environment variable """
+
+    def __init__(
+            self,
+            name: SomeSubstitutionsType,
+            value: SomeSubstitutionsType,
+            **kwargs
```

Comment:
nitpick: indentation is too much here, please just use four spaces

Reply:
Updated the indentation.